ipapython/certdb.py

```diff
@@ -271,7 +283,14 @@ def __enter__(self):
     def __exit__(self, type, value, tb):
         self.close()
 
+    def _check_db(self):
```

Comment:
Pointless note: I wonder if it would be more readable if this was a decorator 🤔

Reply:
Perhaps. On the other hand I'd have to write a nested decorator because I need to access self inside the decorator. At the end I decided that a simple ``self._check_db`` is simpler than a fancy decorator. CPython's io, socket, and ssl module use the same approach because it's also slightly faster.